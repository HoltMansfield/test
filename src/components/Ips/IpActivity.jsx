import React from 'react'
import styled from 'styled-components'


const Grid = styled.div`
  padding-bottom: 10px;
`
const IpColumn = styled.span`
  float: left;
  width: 95px;
`
const LastConnectedColumn = styled.span`
  font-style: italic;
`

export default function ({ ip }) {
  if (!ip) return null

  return (
    <Grid>
      <IpColumn>{ ip.ip }</IpColumn>
      <LastConnectedColumn>Last Connected at: { ip.timestamp }</LastConnectedColumn>
    </Grid>
  )
}
