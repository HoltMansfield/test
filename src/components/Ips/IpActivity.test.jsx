import React from 'react'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import IpActivity from './IpActivity'


it('renders correctly', () => {
  const props = {
      ip: {
        ip: '192.168.0.1',
        timestamp: new Date('January 1, 1990')
    }
  }
  const wrapper = shallow(<IpActivity {...props} />)

  expect(toJson(wrapper)).toMatchSnapshot()
})
