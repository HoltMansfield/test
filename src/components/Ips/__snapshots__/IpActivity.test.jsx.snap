// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div>
  <styled.span>
    192.168.0.1
  </styled.span>
  <styled.span>
    Last Connected at: 
  </styled.span>
</styled.div>
`;
